src/Symfony/Component/Validator/Constraints/Bic.php

```diff
@@ -36,4 +37,14 @@ class Bic extends Constraint
     );
 
     public $message = 'This is not a valid Business Identifier Code (BIC).';
+
+    public function __construct($options = null)
+    {
+        // @deprecated since Symfony 4.2, throw a LogicException in 5.0
```

Comment:
can you please add the future exception in a commented line of code?

Reply:
done!